core/embed/trezorhal/stm32u5/stm32u5xx_hal_sd.c

```diff
@@ -0,0 +1,4083 @@
+// clang-format off
+/**
+  ******************************************************************************
```

Comment:
I think it's time to create our own clone of STM32CubeU5 repository a start patching it. This is the third example, we needed to modify ST code.

Reply:
done. only change in the hal drivers so far is the SD card modification. 

https://github.com/trezor/trezor-firmware/pull/3422/commits/b093a28aa9cd49dd2bb156dab2a498e6593f2b0b